src/lib/isPostalCode.js

```diff
@@ -31,6 +31,7 @@ const patterns = {
   IE: /^[A-z]\d[\d|w]\s\w{4}$/i,
   IL: fiveDigit,
   IN: /^((?!10|29|35|54|55|65|66|86|87|88|89)[1-9][0-9]{5})$/,
+  IR: /^\b(?!(\d)\1{3})[13-9]{4}[1346-9][013-9]{5}\b$/,
```

Comment:
Add the additional tests for this. What was the bug?

Reply:
OK, I'm working on it.
Nothing actually, a conflict happened with an additional semicolon at the end of the object.